packages/neon/neon/lib/src/blocs/user_statuses.dart

```diff
@@ -61,15 +58,15 @@ class UserStatusesBloc extends InteractiveBloc implements UserStatusesBlocEvents
       _updateStatus(username, Result.loading());
 
       UserStatusPublicStatus? data;
-      if (_account.username == username) {
-        final isAway =
-            _platform.canUseWindowManager && (!(await windowManager.isFocused()) || !(await windowManager.isVisible()));
-        final response = await _account.client.userStatus.heartbeat(
+      if (_client.username == username) {
+        final isAway = NeonPlatform.instance.canUseWindowManager &&
+            (!(await windowManager.isFocused()) || !(await windowManager.isVisible()));
```

Comment:
:confounded:

Reply:
hm?

Reply:
I don't like those inline awaits.

Both should be assigned to a variable that's then used in the assignment